test/e2e/auth/node_authz.go

```diff
@@ -174,6 +174,12 @@ var _ = SIGDescribe("[Feature:NodeAuthorizer]", func() {
 		}
 		ginkgo.By(fmt.Sprintf("Create node foo by user: %v", asUser))
 		_, err := c.CoreV1().Nodes().Create(context.TODO(), node, metav1.CreateOptions{})
+		defer func() {
+			err := f.ClientSet.CoreV1().Nodes().Delete(context.TODO(), node.Name, metav1.DeleteOptions{})
+			if err != nil {
+				framework.Logf("Failed to get delete node %v, err: %v", node, err)
```

Comment:
In the expected test path, this `Delete()` call is always failed.
I don't think it is worth to put any log which contains `Failed ...` even but the log level is not Error.

Reply:
Done